code/modules/antagonists/cult/runes.dm

```diff
@@ -284,11 +284,16 @@ structure_check() searches for nearby cultist structures required for the invoca
 	if(!C)
 		return
 
+	if(ispAI(sacrificial))
+		for(var/M in invokers)
+			to_chat(M, span_cultitalic("You don't think this is what Nar'Sie had in mind when She asked for blood sacrifices..."))
```

Comment:
```suggestion
			to_chat(M, span_cultitalic("You don't think this is what Nar'Sie had in mind when It asked for blood sacrifices..."))
```

My first idea was just to have randomly selected pronouns, but maybe this is the actual answer.

Reply:
Nar'Sie has always been a female

Reply:
everything in code references Narsie as female, an example is the Narsie plushie uses the word "Elder Goddess" and the plushies gender is set to female